packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -778,7 +814,7 @@ describe('postgresql-multi-schema', () => {
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
     expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
     expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
-  })
+  }, 50_000)
```

Comment:
Why is this skipped now?

Reply:
Heck, this was unintentional, I had CMD+F'd the "skip" string in https://github.com/prisma/prisma/pull/16766/files without realizing `DbPull.test.ts` was compressed due to the diff being too large

Reply:
Thankfully the test passes anyway